docs/10-clusters.md

```diff
@@ -86,6 +86,14 @@ aztk spark cluster run --id <your_cluster_id> "<command>"
 
 The command is executed through an SSH tunnel.
 
+### Run a command on a specific node in the cluster
+To run a command on all nodes in the cluster, run:
+```sh
+aztk spark cluster node-run --id <your_cluster_id> --node-id <your_node_id> "<command>"
```

Comment:
exec or node-exec? Stays more in line with docker exec

Reply:
There is already a `cluster run` so I figured `cluster node-run` made the most sense here.